BUILDING.md

```diff
@@ -65,22 +72,29 @@ The build output will be in the directory you specified with `do config`.
 
 ### Install Android SDK
 * Unzip the [Android SDK](https://dl.google.com/android/repository/tools_r25.2.3-macosx.zip) to a directory of your choosing.
+  * Instead, if you have Homebrew, you can install via `brew install android-sdk` and the default installation location would be `$HOMEBREW_PREFIX/opt/android-sdk`.
```

Comment:
I was thinking about including the `brew config | grep ... | cut ...` command part earlier. But I feel it's too verbose. Just mentioning `brew config` sounds better.